sql-statements/sql-statement-create-index.md

```diff
@@ -10,37 +10,38 @@ aliases: ['/docs-cn/v2.1/sql-statements/sql-statement-create-index/','/docs-cn/v
 
 ## 语法图
 
-**CreateIndexStmt:**
+```ebnf+diagram
+CreateIndexStmt ::=
+    'CREATE' IndexKeyTypeOpt 'INDEX' IfNotExists Identifier IndexTypeOpt 'ON' TableName '(' IndexPartSpecificationList ')' IndexOptionList IndexLockAndAlgorithmOpt
 
-![CreateIndexStmt](/media/sqlgram/CreateIndexStmt.png)
+CreateIndexStmtUnique ::=
+    UNIQUE?
```

Comment:
Same as the previous comment.

Reply:
i guess this should be inlined into `IndexType`. BTW `'UNIQUE'` not `UNIQUE`.